ONSdigital/eq-author-app
# Push to fix the existing issue. Causes EAR-1397 to rehappen though.

### Context of PR ###

This is a fix to the duplicating issue that's causing problems. This does reintroduce the bug in `EAR-1397`. But this gets around the duplication bug. 

### How to review ###

- Create a question and answer.
- Duplicate the question.
- Delete the answer in the duplication.
- Check that the answer isn't deleted in the original question.

### What to do after everything is green ###

    *  Bring this branch up-to-date with Origin/Master
    *  If there are a lot of commits or some are not helpful to read, squash them down
    *  Click the Merge button on this pull request
    *  Does this change mean the Capability examples questionnaire should be updated?
    *  Move the Jira ticket for this task into the next stage of the process
